firmware/shared/src/hw/hw_runTimeStat.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <stdio.h>
+#include "main.h"
+#include "app_utils.h"
```

Comment:
similar with these three

Reply:
do you need them in the header file? Or can you include it as part of hw_runTimeStat.c?